utils/fill_nines.h

```diff
@@ -1,8 +1,11 @@
 #ifndef UTILS_FILL_NINES_H_
 #define UTILS_FILL_NINES_H_
 
+#include <stddef.h>
+#include <stdlib.h>
+#include <string.h>
```

Comment:
Are these really need to include necessary? I tested on node0 and it seems no need.